src/app/component/CheckoutShippingStep/CheckoutShippingStep.component.js

```diff
@@ -144,8 +145,9 @@ class CheckoutShippingStep extends Component {
 
     renderField(id, overrideStateValue) {
         const { [id]: stateValue } = this.state;
+        const isSelect = id === 'country_id';
         const {
-            type = 'text',
+            type = isSelect ? 'select' : 'text',
```

Comment:
Is this needed? You were able to overwrite type in fieldMap?

Reply:
Resolved.